internal/cfgplugins/staticnexthop.go

```diff
@@ -154,6 +170,59 @@ func NextHopGroupConfig(t *testing.T, dut *ondatra.DUTDevice, traffictype string
 
 }
 
+// NextHopGroupConfigScale configures the interface next-hop-group config.
+func NextHopGroupConfigScale(t *testing.T, dut *ondatra.DUTDevice, sb *gnmi.SetBatch, encapparams OCEncapsulationParams, ni *oc.NetworkInstance, params StaticNextHopGroupParams) *gnmi.SetBatch {
+	if deviations.NextHopGroupOCUnsupported(dut) {
+		switch dut.Vendor() {
+		case ondatra.ARISTA:
+			ipPrefix := strings.Join(strings.Split(strings.Split(encapparams.GRETunnelDestinationsStartIP, "/")[0], ".")[:3], ".")
```

Comment:
You will have much better validation if  you use `net.ParseCIDR` to validate and extract the prefix.

```suggestion
			n, _, err := net.ParseCIDR(encapparams.GRETunnelDestinationsStartIP)
			if err != nil {
				t.Fatalf("invalid IP address %q provided in GRETunnelDestinationsStartIP - err: %v", err)
			}
			ipPrefix := n.String()
```